### DISCUSSION TITLE
Jellyfin-vue trying to access using https on http jellyfin server

### DISCUSSION THREAD

**Discussion by bagusnl** [jellyfin/jellyfin-vue#2114] [Questions] [answered]
Hi, 
I haven't been able to try out jellyfin-vue outside of just looking at dashboards. Then I noticed when I looked at the DevTools console that the jellyfin-vue client (running in docker, accessed to local IP using http) is trying to access the jellyfin server (running on baremetal, accessed to local IP using http) with https, causing ERR_SSL_PROTOCOL_ERROR.

Details:
Jellyfin Server:
 - Version 10.8.10
 - Ubuntu Server Jammy
 - AMD A4-3300M Laptop, 4GB RAM

Jellyfin-Vue:
 - Commit https://github.com/jellyfin/jellyfin-vue/commit/c37a0e39f0371341ec9ac9ad2507487ce622eeda
 - Running in Docker (same bare server as the main Jellyfin server)

Client:
 - Windows 11
 - Edge Version 119.0.2151.12 (Official build) stable app, beta channel (64-bit)
 - Direct LAN
![image](https://github.com/jellyfin/jellyfin-vue/assets/28079733/14267344-8507-400a-b0b7-655c249652ce)

**Comment by ferferga** [comment, marked as answer]
This is expected: The SDK tries to bind to all possible addresses when adding a server when no protocol is specified, hence errors are printed in console.

If I recall correctly, I think we tried to remove those, but we can't.

**Comment by bagusnl** [comment]
Got it, adding `http://` specifically in the server selector fixed the playback issue. Though, some of the SSL error still occurs (somehow fetching to /System/Info/Public) but the playback is fine. 
Thank you.
